### PR TITLE
Display Ghosts, Food, Energizer correctly

### LINKED ISSUE #102: Use image instead of Rect for the Entities
- [x] The 4 ghosts should be differently colored as in the classic Pacman game. Use a separate 5th image to represent the state of afraid ghosts.
- [x] The `food` and `energizer` need to be drawn as the `Circle` of different radii.

### LINKED ISSUE #103: Make the Canvas background black
Currently, each free grid is drawn separately and on each re-render. Making the canvas background `black` prevents the need to do this. Reduces CPU usage. Related to #51

### PR DESCRIPTION
**Pull request by shivangg**
Using `Image`s for ghosts and `Circle`s for the food and energizer.
Makes the canvas background black.

Resolves #102 , Resolves #103 

**Before:**
image

**Now:** 
image

### CODE REVIEW

**Comment by shivangg** [comment]
**Ghosts when in afraid state:**
afraidGhosts

File: client/src/App.css [hunk 1]
```diff
@@ -7,11 +7,15 @@ body{
   position: absolute;
   top: 50%;
   left: 50%;
-  transform: translate(-50%, -50%);
+	transform: translate(-50%, -50%);
```
Comment (nimish-gupta) [inline client/src/App.css]: Indentation

File: client/src/App.css [hunk 2]
```diff
@@ -7,11 +7,15 @@ body{
...
+}
+
+.konvajs-content {
+	background: black;
 }
 
 .App {
-  text-align: center;
+	text-align: center;
```
Comment (nimish-gupta) [inline client/src/App.css]: Indentation?

File: client/src/components/PacmanGame/Cell.jsx [hunk 3]
```diff
@@ -14,9 +15,43 @@ class Cell extends React.Component {
...
+
+    if (!color) {
+      return null;
+    }
+
+    if (entity === 'ghost' || entity === 'scatterGhost') {
+      const image = new window.Image(gridSize, gridSize);
+      image.src = ghostImages[ghostIndex];
+      if (entity === 'scatterGhost') {
+        const [, , , , afraidGhost] = ghostImages;
```
Comment (nimish-gupta) [inline client/src/components/PacmanGame/Cell.jsx]: Why not use indexing to get value?

**Review by nimish-gupta — CHANGES_REQUESTED** [review]

File: client/src/App.css (hunk 2 above)
Comment (shivangg) [reply client/src/App.css]: Converting tabs to spaces.
Making a new commit to convert whole file tabs to spaces.

File: client/src/App.css (hunk 1 above)
Comment (shivangg) [reply client/src/App.css]: Converting tabs to spaces.

**Review by hassan23 — CHANGES_REQUESTED** [review]
Commit name should be **refactor** not **fefactor**

File: client/src/components/PacmanGame/Cell.jsx (hunk 3 above)
Comment (shivangg) [reply client/src/components/PacmanGame/Cell.jsx]: Done

**Review by hassan23 — APPROVED** [review]
LGTM

**Comment by shivangg** [comment]
> Code Climate has analyzed commit [eed406a](https://github.com/pesto-students/batch5-pacman/commit/eed406af00baf58b7f2c380815bbf0d0f02b755d) and detected **6 issues** on this pull request.
> 
> Here's the issue category breakdown:
> Category 	Count
> Duplication 	6
> 
> View more on [Code Climate](https://codeclimate.com/github/pesto-students/batch5-pacman/pull/104).

Some issues like the `board` array duplication are because currently it's both on the server and the client. Moving functions from client to logic gradually and is out of the scope of this PR.

**Review by nimish-gupta — APPROVED** [review]

**Pull request merged**